gravitee-apim-rest-api/gravitee-apim-rest-api-service/pom.xml

```diff
@@ -88,10 +88,15 @@
 			<groupId>io.gravitee.integration</groupId>
 			<artifactId>gravitee-integration-api</artifactId>
 		</dependency>
-        <dependency>
-            <groupId>io.gravitee.scoring</groupId>
-            <artifactId>gravitee-scoring-api</artifactId>
-        </dependency>
+    <dependency>
+        <groupId>io.gravitee.scoring</groupId>
+        <artifactId>gravitee-scoring-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.gravitee.spec.gen.api</groupId>
+      <artifactId>gravitee-spec-gen-api</artifactId>
+      <version>${gravitee-spec-gen-api.version}</version>
```

Comment:
Could you check indentation here. Tab vs Space
And declare the dependency with version in the apim-bom. hey you just need to keep dependency without `<version>`